pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md

```diff
@@ -7,3 +7,11 @@ excerpt: Workspace Catalog Applications
 ---
 <!-- TODO: settle down the link target to custom applications -->
 Catalog applications are any third-party or open source applications that appear in the Catalog. These applications can be provided by D2iQ for use in your environment.
+
+## Workspace catalog applications
+
+| NAME                          | APP ID                |
+| ----------------------------- | --------------------- |
+| kafka-operator-0.20.0         | kafka-operator        |
```

Comment:
we should add kafka with a lowercase to our accepted words list. Or we can ignore this for now.

Reply:
Added

Reply:
Yes Colin has added that to `develop` branch this morning so that I can make this commit.
However, right now the GH Actions is looking at `main` for the accepted words so it appears to be broken on Vale but actually not.